src/django_components/component_registry.py

```diff
@@ -268,6 +270,39 @@ def clear(self) -> None:
         self._registry = {}
         self._tags = {}
 
+    def _register_to_library(
+        self,
+        comp_name: str,
+        component: Type["Component"],
+    ) -> ComponentRegistryEntry:
+        # Lazily import to avoid circular dependencies
+        from django_components.templatetags.component_tags import do_component_tag_block, do_component_tag_inline
+
+        formatter = get_component_tag_formatter()
+
+        # TODO - Allow None here?
+        # register the inline tag
+        inline_tag = formatter.safe_format_inline_tag(comp_name)
```

Comment:
Currently we expect that `TagFormater` WILL define tags for both block and inline variants. If we want to allow `TagFormatters` that would for example implement only the block variant (like discussed here https://github.com/EmilStenstrom/django-components/pull/566#discussion_r1703654170), we could refactor this code to either allow :
- `format_inline_tag` not to be defined,
- or (better) allow `format_inline_tag` to return `None`, which would mean "this TagFormatter does not implement inline tags"